.github/workflows/bdd.yml

```diff
@@ -0,0 +1,63 @@
+name: BDD Tests
+on:
+  push:
+    # branches:
+    #   - master
+    # paths-ignore:
+    # - 'README.md'
+    # - '.vscode/**'
+    # - '**.md'
+  pull_request:
+    branches:
+      - master
+    paths-ignore:
+    - 'README.md'
+    - '.vscode/**'
+    - '**.md'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    container: rofrano/pipeline-selenium:latest
+
+    services:
+      postgres:
+        image: postgres
+        env:
+          POSTGRES_PASSWORD: postgres
+          POSTGRES_DB: postgres
+        ports:
+          - 5432:5432
+        # Set health checks to wait until postgres has started
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
Please add a name for this step, to be consistent with tdd.yml.

Reply:
Done.